### PR TITLE
ja: Make docs/concepts/architecture/controller.md follow v1.19 of the original text

### PR DESCRIPTION
**Pull request by KobayashiD27**
update ja/docs/concepts/architecture/controller.md
#26854

### CODE REVIEW

**Comment by inductor** [comment]
@KobayashiD27 Base branch is incorrect. Please use `dev-1.19-ja.1` .

**Comment by KobayashiD27** [comment]
> @KobayashiD27 Base branch is incorrect. Please use `dev-1.19-ja.1` .

Thank you for pointing out.
Sorry, I took mistake. I changed the base branch.

File: content/ja/docs/concepts/architecture/controller.md [hunk 1]
```diff
@@ -50,7 +50,11 @@ Jobとは対照的に、クラスターの外部に変更を加える必要が
 
 外部の状態とやりとりをするコントローラーは、目的の状態をAPIサーバーから取得した後、外部のシステムと直接通信し、現在の状態を目的の状態に近づけます。
 
-(クラスター内のノードを水平にスケールさせるコントローラーが実際に存在します。詳しくは、[クラスターのオートスケーリング](/docs/tasks/administer-cluster/cluster-management/#cluster-autoscaling)を読んでください。)
+(クラスター内のノードを水平にスケールさせる[コントローラー](https://github.com/kubernetes/autoscaler/)が実際に存在します。)
+
+ここで重要な点は、コントローラーが目的の状態を実現するために変更を加えてから、現在の状態をクラスターのAPIサーバーに報告することです。 他の制御ループは、その報告されたデータを監視し、独自のアクションを実行できます。
```
Comment (inductor) [inline content/ja/docs/concepts/architecture/controller.md:55]:
```suggestion
ここで重要な点は、コントローラーが目的の状態を実現するために変更を加えてから、現在の状態をクラスターのAPIサーバーに報告することです。他の制御ループは、その報告されたデータを監視し、独自のアクションを実行できます。
```

File: content/ja/docs/concepts/architecture/controller.md [hunk 2]
```diff
@@ -50,7 +50,11 @@ Jobとは対照的に、クラスターの外部に変更を加える必要が
...
+
+サーモスタットの例では、部屋が非常に寒い場合、別のコントローラーが霜防止ヒーターをオンにすることもあります。 Kubernetesクラスターを使用すると、コントロールプレーンは、[Kubernetesを拡張して](/ja/docs/concepts/extend-kubernetes/)実装することにより、IPアドレス管理ツールやストレージサービス、クラウドプロバイダーAPI、およびその他のサービスと間接的に連携します。
```
Comment (inductor) [inline content/ja/docs/concepts/architecture/controller.md:57]:
```suggestion
サーモスタットの例では、部屋が非常に寒い場合、別のコントローラーが霜防止ヒーターをオンにすることもあります。Kubernetesクラスターを使用すると、コントロールプレーンは、[Kubernetesを拡張して](/ja/docs/concepts/extend-kubernetes/)実装することにより、IPアドレス管理ツールやストレージサービス、クラウドプロバイダーAPI、およびその他のサービスと間接的に連携します。
```

**Review by inductor — CHANGES_REQUESTED** [review]
I don't think these blanks are necessary. Please check there's any other too.

**Comment by KobayashiD27** [comment]
> I don't think these blanks are necessary. Please check there's any other too.

Thanks. I checked for unnecessary blanks, but nothing else.

**Pull request merged**